src/loaders/utils/GetTakeoverDetails.ts

```diff
@@ -0,0 +1,57 @@
+import { isServerSide } from 'utils/getEnv';
+import { getArticleVideoServiceBaseUrl } from 'utils/getPandoraDetails';
+import { trackYoutubeTakeoverDetails, YoutubePlayerTrackingProps } from 'youtube/players/shared/youtubeTrackingEvents';
+import { getMediaWikiConfigDetails } from 'loaders/utils/GetMediaWikiConfigDetails';
+import { TakeoverDetails, TakeoverResponse } from 'loaders/types';
+
+function getTakeoverUrl(wikiId: string): string {
+	const articleVideoBaseUrl = getArticleVideoServiceBaseUrl();
+	if (articleVideoBaseUrl.length === 0) {
+		return articleVideoBaseUrl;
+	}
+	if (!wikiId) {
+		return '';
+	}
+	return `${articleVideoBaseUrl}takeover/v1/takeover-mappings/${wikiId}`;
+}
+
+export async function getTakeoverDetails({ deviceType }: YoutubePlayerTrackingProps) {
+	const { wikiId, isTier3Wiki } = getMediaWikiConfigDetails();
+	if (!wikiId || isTier3Wiki) {
+		return null;
+	}
+
+	const response: Response = await fetch(getTakeoverUrl(wikiId));
+	const dataArray: TakeoverResponse[] = await response.json();
+	const takeoverDetails: TakeoverDetails = { videoId: null, type: null };
+
+	if (dataArray.length === 0 || !dataArray || isServerSide()) {
```

Comment:
Do you know what's the purpose of this `isServerSide()` check? I think we actually never use the player on the server side. 

Also I noticed that this function is actually broken:
```const isServerSide = () => {
	return typeof window === undefined;
};```
undefined is not wrapped with `'` so it will always return false. IMO we can remove this function and all of the usages. :)